charts/ocis/templates/sharing/deployment.yaml

```diff
@@ -78,6 +78,10 @@ spec:
             - name: SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD
               value: {{ .Values.features.sharing.publiclink.writeableShareMustHavePassword | quote }}
 
+            # TODO: does not exist currenlty. Probably should be there because we also have SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD!?
```

Comment:
```suggestion
            # TODO: does not exist currently. Probably should be there because we also have SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD!?
```